src/main/java/com/spotinst/sdkjava/model/api/admin/account/ApiOrganizationUserGroups.java

```diff
@@ -0,0 +1,122 @@
+package com.spotinst.sdkjava.model.api.admin.account;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiOrganizationUserGroups implements IPartialUpdateEntity {
+    @JsonIgnore
+    private Set<String>                    isSet;
+    private String                         id;
+    private String                         name;
+    private String                         description;
+    private String                         createdAt;
+    private String   usersCount;
+    private List<String> policyNames;
+
+    public ApiOrganizationUserGroups() {
+        isSet = new HashSet<>();
+    }
+
+    public Set<String> getIsSet() {
+        return isSet;
+    }
+
+    public void setIsSet(Set<String> isSet) {
+        this.isSet = isSet;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        isSet.add("id");
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        isSet.add("name");
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        isSet.add("description");
+        this.description = description;
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(String createdAt) {
+        isSet.add("createdAt");
+        this.createdAt = createdAt;
+    }
+
+    public String getUsersCount() {
```

Comment:
Is this a string?

Reply:
It is an integer. Updated both bl and api class.